boilerplate/app/components/checkbox/checkbox.story.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable react-native/no-inline-styles */
```

Comment:
I opted to disable these two rules in stories because I believe they bring value to the codebase but at the same time believe that worrying about these rules when writing a story is a little overboard. I am open to opinions of going one way or the other.